packages/FormElement/src/FormElement.styles.js

```diff
@@ -7,19 +7,23 @@ const FontSizes = {
   large: stylers.fontSize(0),
 };
 
-export const SectionsContainer = styled.div`
-  ${({ isInline }) => isInline && `flex-grow: 1;`}
-`;
+export const SectionsContainer = styled.div(
+  ({ isInline }) => css`
+    ${isInline && `flex-grow: 1;`}
+  `
+);
```

Comment:
There's no point adding `css` here. If you really want it you might want to add it like this
```
export const SectionsContainer = styled.div`
  ${({ isInline }) => isInline && css`flex-grow: 1;`}
`;
```